src/test_fixture/sharing.rs

```diff
@@ -34,3 +36,16 @@ pub fn validate_and_reconstruct<T: Field>(
 
     input.0.left() + input.1.left() + input.2.left()
 }
+
+/// Validates expected result from the secret shares obtained.
+///
+/// # Panics
+/// Panics if the expected result is not same as obtained result. Also panics if `validate_and_reconstruct` fails
+pub fn validate_result_from_shares(expected_result: &[u128], result: &ReplicatedShares) {
```

Comment:
The name of this function in no way indicates that this is only to be used to validate slices / arrays / lists. Let's call it something like `validate_list_of_shares`